src/python/bot/fuzzers/afl/launcher.py

```diff
@@ -310,12 +310,16 @@ def to_strategy_dict(self):
     # The decision on whether or not fast cal should be used is made during
     # fuzzing. This function is expected to be called whe preparing for fuzzing.
     assert self.fast_cal == strategies.FastCal.NOT_SET
+    strategies_dict = {}
+
+    if self.generator_strategy == engine_common.Generator.RADAMSA:
```

Comment:
Do we need the FastCal ones here too ?

Reply:
See line 312 and https://github.com/google/clusterfuzz/pull/2211#discussion_r564941258